managed/src/main/java/com/yugabyte/yw/controllers/UniverseController.java

```diff
@@ -99,7 +99,6 @@ public UniverseController(YBClientService service) {
    * @return true if universe already exists, false otherwise
    */
   // TODO(.*endra):  This method is buggy. The javadoc does not match impl.
-  //  Replace findByName with findByName_Corrected when UI is fixed to expect opposite.
   @Deprecated
   public Result findByName(UUID customerUUID, String universeName) {
```

Comment:
- Delete this method.
- Rename findByName_Corrected to findByName
- Fix all the tests.

Reply:
1, 2 updated. 3rd already covered. Have updated the existing test case for find API and added 2 test case for check_unique as well.